conda_forge_tick/migrators.xsh

```diff
@@ -335,9 +335,11 @@ class Compiler(Migrator):
 
     rerender = True
 
+    compilers = ['toolchain', 'gcc']
+
     def filter(self, attrs):
         conditional = super().filter(attrs)
-        return conditional or 'toolchain' not in attrs.get('req', [])
+        return conditional or not any(x in attrs.get('req', []) for x in self.compilers)
```

Comment:
We should not do this for R recipes as there are some conda-build issue we should figure out